libs/hdf-converters/src/conveyor-mapper.ts

```diff
@@ -142,18 +155,21 @@ function preprocessObject(
   return newSections;
 }
 
+/*
+Builds everything under profiles.controls using results.sections data from Conveyor JSON.
+*/ 
 function controlMappingConveyor(): MappedTransform<
   ExecJSON.Control & ILookupPath,
   ILookupPath
 > {
   return {
     id: {path: 'sha256'},
-    title: {path: 'filename'},
-    desc: '',
+    title: {path: 'filename'},  // Should be heur_id from conveyor
```

Comment:
there seems to be a discrepancy between the newly added comment and what attribute we've been using here.  which is correct?